prod/templates/config/asset-hub-ethereum-relay.json

```
@@ -0,0 +1,26 @@
+{
+  "source": {
+    "ethereum": {
+      "endpoint": "ws://127.0.0.1:8546"
+    },
+    "contracts": {
+      "Gateway": null
+    },
+    "channel-id": null,
+    "beacon": {
+      "endpoint": "http://127.0.0.1:9596",
+      "spec": {
+        "slotsInEpoch": 32,
+        "epochsPerSyncCommitteePeriod": 256,
+        "denebForkedEpoch": 269568
+      }
+    }
```

```suggestion
    "channel-id": null
```

The beacon config was only added in the polkadot 1.10.0 version, so we can remove this.